frontend/src/components/UpgradeDialog.tsx

```diff
@@ -0,0 +1,122 @@
+          <DialogTitle>✨ Unlock Alby Pro</DialogTitle>
+          <DialogDescription>
+            Take your Alby Hub experience to the next level
+          </DialogDescription>
+        </DialogHeader>
+        <div className="flex flex-col gap-6 my-4">
+          <div className="space-y-5">
+            <h3 className="font-medium text-lg">Pro Features Include:</h3>
+            <ul className="space-y-3">
+              <li className="flex items-center">
+                <Users className="w-5 h-5 mr-3 text-primary" />
+                <div>
+                  <span className="font-medium">Unlimited Sub-wallets</span>
+                  <p className="text-sm text-muted-foreground">
+                    Share with friends, family, coworkers
+                  </p>
+                </div>
+              </li>
+              <li className="flex items-center">
+                <RefreshCw className="w-5 h-5 mr-3 text-primary" />
+                <div>
+                  <span className="font-medium">Encrypted Remote Backups</span>
+                  <p className="text-sm text-muted-foreground">
+                    Secure LDK wallet backups in the cloud
+                  </p>
+                </div>
+              </li>
+              <li className="flex items-center">
+                <LifeBuoy className="w-5 h-5 mr-3 text-primary" />
+                <div>
+                  <span className="font-medium">Priority Support</span>
+                  <p className="text-sm text-muted-foreground">
+                    Get help faster when you need it
+                  </p>
+                </div>
+              </li>
+              <li className="flex items-center">
+                <Zap className="w-5 h-5 mr-3 text-primary" />
+                <div>
+                  <span className="font-medium">Custom Lightning Address</span>
+                  <p className="text-sm text-muted-foreground">
+                    Create your personalized address
+                  </p>
+                </div>
+              </li>
+              <li className="flex items-center">
+                <Mail className="w-5 h-5 mr-3 text-primary" />
+                <div>
+                  <span className="font-medium">Email Notifications</span>
+                  <p className="text-sm text-muted-foreground">
+                    Stay updated on important activity
+                  </p>
+                </div>
+              </li>
+              {/* 
+              <li className="flex items-center">
+                <Server className="w-5 h-5 mr-3 text-primary" />
+                <div>
+                  <span className="font-medium">Node Monitoring</span>
+                  <p className="text-sm text-muted-foreground">
+                    Keep your node running smoothly
+                  </p>
+                </div>
+              </li>
+              */}
+            </ul>
+          </div>
+        </div>
+        <div className="flex-col gap-1">
+          <Button
```

Comment:
Can we use the `ExternalLinkButton` component instead? buttons that are not proper links are bad UX